tomography_preprocessing/tilt_series_alignment/aretomo/align_tilt_series.py

```diff
@@ -21,6 +21,7 @@ def align_single_tilt_series(
         n_patches_xy: Tuple[int, int],
         alignment_thickness_px: float,
         tilt_angle_offset_correction: bool,
+        gpu_id: str or None,
```

Comment:
this type hint should look like 🙂 
```suggestion
        gpu_id: Optional[str],
```

from python 3.10 you can also do
```
str | None
```
see [PEP604](https://peps.python.org/pep-0604/) for details